_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -15,6 +15,12 @@
 // Load WP_Error for error messages.
 require_once ABSPATH . '/wp-includes/class-wp-error.php';
 
+// Load API endpoint base classes
+require_once JETPACK__PLUGIN_DIR . '/_inc/lib/core-api/class.jetpack-core-api-xmlrpc-consumer-endpoint.php';
+
+// Load API endpoints
+require_once JETPACK__PLUGIN_DIR . '/_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php';
```

Comment:
This class should be loaded only if it's going to be used, in `Jetpack_Core_Json_Api_Endpoints::register_endpoints`.